ui/tests/acceptance/allocation-detail-test.js

```diff
@@ -397,26 +407,62 @@ module('Acceptance | allocation detail', function (hooks) {
   });
 
   test('while an allocation is being restarted, the stop button is disabled', async function (assert) {
-    server.pretender.post('/v1/allocation/:id/stop', () => [204, {}, ''], true);
+    let resolveRestart;
+    const restartRequestPromise = new Promise((resolve) => {
+      resolveRestart = resolve;
+    });
+    server.pretender.put('/v1/client/allocation/:id/restart', () => {
+      return restartRequestPromise.then(() => {
+        return [204, {}, ''];
+      });
+    });
 
-    await Allocation.stop.idle();
+    await Allocation.restart.idle();
+    await Allocation.restart.confirm();
 
-    run.later(() => {
-      assert.ok(Allocation.stop.isDisabled, 'Stop is disabled');
-      assert.ok(Allocation.restart.isDisabled, 'Restart is disabled');
-      assert.ok(Allocation.restartAll.isDisabled, 'Restart All is disabled');
-      server.pretender.resolve(server.pretender.requestReferences[0].request);
```

Comment:
Note: with the changes here, requestReferences was always `[]` during this test.